### DISCUSSION TITLE
US Roaster Corp -> iMac

### DISCUSSION THREAD

**Discussion by UboraCoffeeRoasters** [artisan-roaster-scope/artisan#1773] [Q&A]
Hey there!
I am running a 12kg US Roaster Corp Millenium and I am just trying to connect to the Artisan software. I get the error, "ModBus failed to connect"

Most things I see tell me to download a driver, but I am not sure which driver is correct. Can anyone help us out?
Thankssss

**Comment by poundy** [comment]
Not to point out the obvious, but did you try looking in the machine specific page? https://artisan-scope.org/machines/usroaster/
That points you to a list of all drivers that machines use. The USRC isn't listed, so you fall back to looking at Device Manager (on Windows; other platforms are at your own peril I mean investigation) and plug the roaster in; once you see the serial device appear and it should give you sufficient leads to identify which driver on the list you need. If not, collect screenshots and post back here...
